scheduler/roles/k8s_start_services/tasks/main.yml

```diff
@@ -18,5 +18,11 @@
     - hostvars['127.0.0.1']['k8s_support']
     - "'kube_control_plane' in group_names"
   block:
+    - name: Change containerd runtime
```

Comment:
Place this task in scheduler/roles/k8s_habana_container_runtime/tasks/main.yml

Reply:
@yhluo946 can you please do this change feel free to reach out to @Katakam-Rakesh if you need help